projects/online/online/utils/pe.py

```diff
@@ -1,24 +1,49 @@
 import logging
 from typing import TYPE_CHECKING
 
-import bilby
-import healpy as hp
 import lal
-import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
 import torch
-from amplfi.train.testing import nest2uniq
-from astropy import io, table
-from astropy import units as u
+from amplfi.train.data.utils.utils import ParameterSampler
+from amplfi.train.priors import precessing_cbc_prior
+from amplfi.utils.result import AmplfiResult
+from ml4gw.distributions import Cosine
+from torch.distributions import Uniform
+
+torch.set_num_threads(1)
```

Comment:
This may not be needed anymore given that you set `export MKL_NUM_THREADS=1 , export OMP_NUM_THREADS=1`